.changeset/wild-dancers-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+"@powersync/react-native-quick-sqlite": minor
```

Comment:
We should probably revert the package name to `@journeyapps/react-native-quick-sqlite` if we want to update this in the PowerSync SDK.